keyboards/keebio/quefrency/rev6/rules.mk

```diff
@@ -0,0 +1,3 @@
+SERIAL_DRIVER = usart
+EEPROM_DRIVER = wear_leveling
+WEAR_LEVELING_DRIVER = embedded_flash
```

Comment:
This is the default behaviour.
```suggestion
```

Reply:
Hmm, it failed to compile either of those lines

Reply:
Odd, pretty sure I'm interpreting the default correctly? Someone else feel free to chime in.
https://github.com/qmk/qmk_firmware/blob/fe50774cb488f981dfd41c7a5bc5f34dfcd9cb9d/builddefs/common_features.mk#L218-L222

Reply:
For me, compiles fine without those lines.

If it works, :shrug:

Reply:
Removed those lines now that I've rebased onto the latest master with the fix